recipes/mpir/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2.7.2" %}
+
+package:
+  name: mpir
+  version: {{ version }}
+
+source:
+  fn: mpir-{{ version }}.tar.bz2
+  url: http://mpir.org/mpir-{{ version }}.tar.bz2
+  md5: b31aa1ebc01b5fedf4ac43277c03feec
+
+build:
+  number: 0
+  skip: true          # [win and not py35]
+  features:
+    - vc9             # [win and py27]
+    - vc10            # [win and py34]
```

Comment:
I think it can support vc10. At least that is present in the source code I downloaded. Is there a reason this wouldn't work, @isuruf.